content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,242 @@
+Begin receiving the incoming shipment by selecting the :guilabel:`Receipt` smart button on the
+:abbr:`PO (Purchase Order)`. Click :menuselection:`Receive products --> Validate` to move products
+into the warehouse. Since the putaway rule is set to trigger when products arrive in stock, internal
+transfers are automatically created when products arrive in `WH/Stock`. This ensures products
+entering the warehouse are stored in specific locations suitable for their storage needs.
+
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: Receive package of apples and carrots
+
+Verify stock moves
+==================
+
+Confirm whether the putaway rule worked as intended and that the products have been moved to the
+correct location using the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory app --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
 
-.. image:: putaway/putaw5.png
+Select a product, and the drop-down menu displays the locations where the products currently are.
+Select the location drop-down menu to view a list of all stock movements in and out of the location.
+Each of these lines are records of a stock move, called a *stock move line*.
+
+.. image:: putaway/destination-locations.png
    :align: center
+   :alt: Verify stock moves triggered by putaway rule
+
+.. _inventory/putaway/wrong-location:
+
+Trace products in wrong locations
+---------------------------------
+
+To trace products and associated documents for warehouse operations, use the :guilabel:`Product
+Moves` report in :menuselection:`Inventory app --> Reporting --> Product Moves`. View a product's
+warehouse operation history by selecting it from the product drop-down. Each stock move line
+contains details about any movement of stock, such as manufacturing orders, inventory adjustments,
+or internal transfers. Click on a stock move line to view all the information in a comprehensive
+report.
+
+This is useful because :guilabel:`Reference` displays the warehouse operation associated to the
+:guilabel:`Product`. Use the :guilabel:`From` and :guilabel:`To` location and associated
+:guilabel:`Source` documents to trace the product path and identify the type of operation.
+
+.. tip::
+   It is also possible to view the stock moves of a product directly from the product page. Open the
+   desired product page by navigating to :menuselection:`Inventory --> Products --> Products` and
+   select the product. From there, click on the :guilabel:`Product Moves` smart button, then the
+   product drop-down. This displays the stock moves history of the product.
+
+.. _inventory/putaway/stock_moves:
+
+.. example::
+   A putaway rule was purposely set up incorrectly on `Apple`, so the product was moved from
+   `WH/Stock` to `WH/Stock/Shelf 1` instead of the expected location, `WH/Stock/Fruits`. To trace
+   the path the apple moved and identify the cause of the wrong stock move, open the
```

Comment:
I agree that product is more general, but since we're putting this in an example block, I thought it'd be more useful to be as specific as possible (: